deploy/charts/harvester/dependency_charts/multus/templates/daemonset.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "multus.name" . }}-ds-amd64
+  namespace: kube-system
+  labels:
+  {{- include "multus.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+  {{- include "multus.selectorLabels" . | nindent 6 }}
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+    {{- include "multus.selectorLabels" . | nindent 8 }}
+    spec:
+      hostNetwork: true
+      serviceAccountName: {{ include "multus.name" . }}
+      containers:
+        - name: kube-multus
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/entrypoint.sh"]
+          args:
+            - "--multus-conf-file=auto"
```

Comment:
I think we also need to set `--multus-kubeconfig-file-host` here, can you confirm?